src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    <p className={`${person.age ? 'Person__age' : ''}`}>
+      {person.age && `I am ${person.age}`}
+    </p>
+    <p className="Person__partner">
+      {person.isMarried === true &&
+        person.sex === 'm' &&
+        `${person.partnerName} is my wife`}
+      {person.isMarried === true &&
+        person.sex === 'f' &&
+        `${person.partnerName} is my husband`}
+      {person.isMarried === false && 'I am not married'}
```

Comment:
Try to use ternary operator here, to make it easier for understanding
Also would be a good idea to create variable for `wife/husband` partner status and use this variable here